### ISSUE TITLE
unique in deviate is not linked correctly

### ISSUE THREAD

**Issue by huyuwen** [theia-ide/yang-lsp#114]
We find a bug related to unique in deviate. For example,

	list l1 {
		leaf lf1 {
			type string;
		}
	}
	deviation l1 {
		deviate add {
			unique lf1;
		}
	}

lf1 will always cause link error.
It is definitly a bug.

**Comment by JanKoehnlein** [comment]
Yes, I see a link error as well, but for `deviation l1`, while `unique lf1` is linked. Am I missing something?

**Comment by huyuwen** [comment]
Here is the screen-shot from latest yangster
![unique-in-deviate](https://user-images.githubusercontent.com/809513/39564724-ad731054-4eb5-11e8-986d-b1ee00b9b3ad.png)

**Comment by JanKoehnlein** [comment]
Hi Yuwen,

here is my more minimal example:



Strange, isn't it? I'll try to fix both issues.

Best regards
Jan

--
Dr. Jan Köhnlein
jan.koehnlein@typefox.io

TypeFox GmbH
Am Germaniahafen 1
24143 Kiel

Phone: +49 151 17396687
Skype: jankoehnlein
Twitter: jankoehnlein

Sitz: Kiel, Registergericht: Amtsgericht Kiel, HRB 17385
Geschäftsführer: Sven Efftinge, Moritz Eysholdt, Dr. Jan Köhnlein

**Comment by huyuwen** [comment]
Hi Jan,

Any progress on this issue?

**Comment by JanKoehnlein** [comment]
For my scenario, it's a missing `;` after the namespace. Then I get the same error as you. 
Now digging deeper...
